sql/core/src/main/scala/org/apache/spark/sql/execution/Exchange.scala

```diff
@@ -158,7 +158,7 @@ case class Exchange(
         val rdd = if (needToCopyObjectsBeforeShuffle(part, serializer)) {
           child.execute().mapPartitions { iter =>
             val hashExpressions = newMutableProjection(expressions, child.output)()
-            iter.map(r => (hashExpressions(r).copy(), r.copy()))
+            iter.map(r => (hashExpressions(r).copy, r.copy()))
```

Comment:
hm since copy is a verb, it is better to keep the () i think. also we defined copy with ()